src/sentry/integrations/utils/commit_context.py

```diff
@@ -63,6 +64,7 @@ def find_commit_context_for_event_all_frames(
         platform=platform,
         sdk_name=sdk_name,
         extra=extra,
+        organization=Organization.objects.get(id=organization_id),
```

Comment:
is there a risk of this throwing a `Organization.DoesNotExist` exception? I only ask because down the line `convert_stacktrace_frame_path_to_source_path` has a typing of `Organization | None`